haystack/utils/openai_utils.py

```diff
@@ -0,0 +1,132 @@
+"""Utils for using OpenAI API"""
```

Comment:
This is the file I created to accumulate the utility functions we've made for interacting with OpenAI API. We had duplicate code in the `_openai_encoder.py`, `prompt_node.py` and `openai.py` files, which now all load the same functions from this utils file.